src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitOperation;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<FruitOperation> SHOP_STORE = new ArrayList<>();
```

Comment:
Use data structures from the Collection framework that best suit your needs. Since `FruitOperation` seems to represent operations on fruits, you may want to use a Map to store fruit types and their quantities instead of a List of operations.